apps/web/tests/e2e/authenticated/comment.test.ts

```diff
@@ -97,7 +97,7 @@ test.describe('create, edit, and delete comments', () => {
 
     await expect(
       page.getByLabel('Description').getByText(`${editedParentComment} ${parentCodeText}`),
-    ).not.toBeVisible();
+    ).not.toBeVisible({ timeout: 10000 });
```

Comment:
this hurts my soul but welcome to e2e i guess.

is there a global timeout config we can put somewhere so it just does this on any of these?

Reply:
It could be done according to the [docs](https://playwright.dev/docs/test-timeouts#expect-timeout), would this be your preferred method?

Reply:
yeah lets do that instead cause i think we'll end up with a lot of ad hoc timeouts otherwise. thoughts?

Reply:
I personally like the approach of adding the timeouts as needed better, because I feel that sometimes the global timeouts are not working as expected. Although I only set the _test_ timeout before, not the _expect_ timeout, so maybe this config would be more reliable?

I will try that later/tomorrow

Reply:
> I personally like the approach of adding the timeouts as needed better, because I feel that sometimes the global timeouts are not working as expected. Although I only set the _test_ timeout before, not the _expect_ timeout, so maybe this config would be more reliable?
> 
> I will try that later/tomorrow

ok lets merge this. we can address if it becomes more of an issue down the road!